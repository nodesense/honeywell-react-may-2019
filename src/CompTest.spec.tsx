// CompTest.spec.tsx
import React from 'react';
import renderer from 'react-test-renderer';

interface TestProps {
  title: string;
}

function Test(props: TestProps) {
    return (
        <div>
          <p>{props.title}</p>
        </div>
    )
}

it('renders correctly', () => {
  const tree = renderer
    .create(<Test title="Hello"></Test>)
    .toJSON();

  expect(tree).toMatchSnapshot();

});


it('renders correctly', () => {
  const tree = renderer
    .create(<Test title="React"></Test>)
    .toJSON();

  expect(tree).toMatchSnapshot();
  
});