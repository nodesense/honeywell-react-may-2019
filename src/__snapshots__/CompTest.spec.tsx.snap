// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Hello
  </p>
</div>
`;

exports[`renders correctly 2`] = `
<div>
  <p>
    React
  </p>
</div>
`;
